snyk/snyk-python-plugin
# test: add tox config for local testing

#### What does this PR do?

Current development documentation suggest dependencies are installed directly on the system without using a virtualenv, against a single python version. Both of these problems can be solved with `tox`.

#### Where should the reviewer start?

`tox.ini`

#### How should this be manually tested?

Clone a fresh copy of the repo. Run `pip install tox`. Running `tox` will now handle dependency install and test running.

This process should work in or out of a virtualenv.

#### Any background context you want to provide?

`tox` acts as a local replacement for the Travis CI build matrix, handling language and dependency installation.

#### What are the relevant tickets?


#### Screenshots


#### Additional questions
